crest/Assets/Crest/Crest/Scripts/Underwater/UnderwaterRenderer.Effect.cs

```diff
@@ -57,21 +86,28 @@ void OnPreRenderUnderwaterEffect()
 
             RenderTextureDescriptor descriptor = XRHelpers.GetRenderTextureDescriptor(_camera);
             descriptor.useDynamicScale = _camera.allowDynamicResolution;
+            // PERF: Auto resolves which triggers more resolves than necessary.
+            descriptor.SetMSAASamples(_camera);
+            descriptor.bindMS = false;
+
             // Format must be correct for CopyTexture to work. Hopefully this is good enough.
             if (_camera.allowHDR)
             {
                 descriptor.colorFormat = RenderTextureFormat.DefaultHDR;
             }
 
             var temporaryColorBuffer = RenderTexture.GetTemporary(descriptor);
+            temporaryColorBuffer.name = "Crest Temporary Color";
```

Comment:
not sure if it makes sense to add "Underwater" to this name?

Reply:
Not really as it is before the underwater is applied. It serves the same purpose as the color texture Unity provides in HDRP/URP. I've renamed it as per other textures: _CrestCameraColorTexture.

Reply:
ok sure